recipes/Polycircles/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "polycircles" %}
+{% set version = "0.3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5e7a1872e88694bbaab98c97912e94babb60edb4be14f84c248dbe11960ac9e3
+
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - geographiclib
+  run:
+    - python
+    - geographiclib
+
+test:
+  imports:
+    - polycircles
+  requires:
+    - geopy >=0.99
+    - nose >=1.3.0
+    - simplekml >=1.2.3
+
+about:
+  home: "http://polycircles.rtfd.org"
+  license: "MIT"
```

Comment:
We need a licence file to comply with the MIT licence. See the example for how to specify it. If it’s not in the source tarball you can also include it in the recipe directory

Reply:
@chrisburr Resolved the issues, thank you!